src/main/java/com/breskul/bring/annotations/Bean.java

```diff
@@ -0,0 +1,24 @@
+package com.breskul.bring.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Indicates that a method produces a bean which will be stored and managed in Bring container.
+ *
+ */
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Bean {
+
+  /**
+   * The name of this bean. If left unspecified
+   * the name of the bean is the name from annotated method.
+   */
+  String name() default "";
```

Comment:
I think it would be better to add method `value` not `name`. If there is just one element named `value`, then the name can be omitted and we can use `@Bean("myBean")` not `@Bean(name="myBean")`.

Reply:
Agree. Updated.

Reply:
Let's do the same to @Component and @Configuration

Reply:
Done.